ucx.spec.in

```diff
@@ -1,4 +1,7 @@
 %{!?configure_options: %global configure_options %{nil}}
+%bcond_with cma
+%bcond_without knem
```

Comment:
Can we disable `knem` and `xpmem` by default? I don't see such packages in Fedora, build on F28 failed.

Reply:
this is done here - "bcond_without" means disable by default

Reply:
Hm. As I understand `bcond_without` introduce `--without` option making it is ON by default. From here: http://rpm.org/user_doc/conditional_builds.html.